### PR TITLE
add partner feeds for other page types

### PR DESCRIPTION
**Pull request by asmega**
e.g. feeds for videos, news etc

### CODE REVIEW

File: app/controllers/api/content_controller.rb [hunk 1]
```diff
@@ -23,13 +23,13 @@ def preview
...
     def published
-      pages = current_site.pages.published
+      pages = current_site.pages.published.joins(:layout).where(comfy_cms_layouts: { identifier: params[:page_type].singularize })
 
       render json: pages, each_serializer: PageFeedSerializer
     end
 
     def unpublished
-      pages = current_site.pages.unpublished
+      pages = current_site.pages.unpublished.joins(:layout).where(comfy_cms_layouts: { identifier: params[:page_type].singularize })
```
Comment (tomas-stefano) [inline app/controllers/api/content_controller.rb]: WE could have some scope like by page type. Wdyt?

**Comment by asmega** [comment]
done. i've added a scope

**Comment by asmega** [comment]
my fault. fixed now

**Comment by tomas-stefano** [comment]
lgtm :+1:

**Pull request merged**